examples/async-universal/server/server.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-console, no-use-before-define */
```

Comment:
Let’s not add per-file ESLint exceptions. These rules exist in the rule config, so just disabling them in a file begs the question why we use that config at all. I would suggest complying with `no-use-before-define` (we do so elsewhere) and moving `no-console` `disable` + `enable` pair specifically to the line where it is being used. (I’m not sure if was an issue in the existing examples as well, but we definitely shouldn’t introduce more per-file rule overrides.)